src/templates/vue/src/main.js

```diff
@@ -17,7 +17,6 @@ const { VUE_APP_AUTH0_DOMAIN, VUE_APP_AUTH0_CLIENT_ID } = process.env
 const app = createApp(App)
 <% if(isStore){ %> .use(store)  <% } %>
 .use(router)
-.use(Vuex)
```

Comment:
why removed Vuex, when the check is of isStore?

Reply:
Because we don't need to use Vuex like this in Vue3.
Used **use(store)** instead of that